erpnext/patches/v7_0/update_status_of_zero_amount_sales_order.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	for data in frappe.get_all('Sales Order', fields = ["name"], filters = [["docstatus", "=", "1"], ["grand_total", "=", "0"]]):
+		print data.name
+		sales_order = frappe.get_doc('Sales Order', data.name)
+		sales_order.set_status(update_modified = False)
```

Comment:
sales_order.set_status(update=True, update_modified = False)

Remove the next line